fail2ban/tests/files/logs/nginx-bad-request

```diff
@@ -20,4 +20,7 @@
 7.8.9.10 - root [20/Jan/2015:01:17:07 +0100] "CONNECT 123.123.123.123 HTTP/1.1" 400 162 "-" "-" "-"
 
 # failJSON: { "time": "2014-12-12T22:59:02", "match": true , "host": "2.5.2.5" }
-2.5.2.5 - tomcat [12/Dec/2014:22:59:02 +0100] "GET /cgi-bin/tools/tools.pl HTTP/1.1" 400 162 "-" "-" "-"
\ No newline at end of file
+2.5.2.5 - tomcat [12/Dec/2014:22:59:02 +0100] "GET /cgi-bin/tools/tools.pl HTTP/1.1" 400 162 "-" "-" "-"
+
+# failJSON: { "time": "2014-12-12T22:59:02", "match": true , "host": "192.168.144.1" }
+192.168.144.1 - - [12/Dec/2014:22:59:02 +0100] "GET /adsf HTTP/1.1" 404 118 "-" "-" "-"
```

Comment:
As already said, 404 is not bad request, so no chance by default for that.
However if you'd implement above suggested enhancement (with code) this test may remain, but would expect an option, for instance:
```diff
+ filterOptions: [{"code": "(?!401)[45]\d{2}"}]
  # failJSON: ...
  ... 404 ...
```